src/SPC/builder/unix/library/imap.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPC\builder\unix\library;
+
+use SPC\exception\FileSystemException;
+use SPC\exception\RuntimeException;
+use SPC\exception\WrongUsageException;
+use SPC\store\FileSystem;
+use function Safe\xdiff_file_patch_binary;
```

Comment:
I don't see any usage of this.

Reply:
woopsie, I considered replacing the xdiff_file_patch method from the xdiff extension, but felt like introducing that requirement was not my decision to make